lib/train-winrm/transport.rb

```diff
@@ -79,6 +79,11 @@ class Transport < Train.plugin(1) # rubocop:disable Metrics/ClassLength
       option :client_key , default: nil
       option :client_key_pass , default: nil
 
+      # new socks proxy options
+      option :socks_proxy, default: nil
+      option :socks_user, default: nil
+      option :socks_password, default: nil
```

Comment:
Where are the options `socks_user` and `socks_password` used in the functionality? Are we just extending them now for future implementations?

Reply:
Oops I have not pushed that change .. let me do that